gfx.c

```diff
@@ -1002,6 +1005,24 @@ void ReadNtrCell_CEBK(unsigned char * restrict data, unsigned int blockOffset, u
         }
     }
 
+    if (options->vramTransferEnabled)
+    {
+        offset = blockOffset + 0x08 + vramTransferOffset;
+        
+        // first 2 dwords are max size and offset, offset *should* always be 0x08 since the transfer data list immediately follows this
+        options->vramTransferMaxSize = data[offset] | (data[offset + 1] << 8) | (data[offset + 2] << 16) | (data[offset + 3] << 24);
+        offset += 0x08;
+
+        // read 1 VRAM transfer data block for each cell (this is an assumption based on the NCERs I looked at)
```

Comment:
only thing I wanted to note on this is this part, the G2d spec indicates `pCellTransferDataArray` is an array (clear from the name) but does not specify the size or if there's 1 element per cell. i made an assumption for this